source/conf.py

```diff
@@ -16,6 +16,8 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import sphinx_rtd_theme
```

Comment:
Is this really necessary? I would have expected just adding it as a dependency would be enough.